docs/DeveloperGuide.adoc

```diff
@@ -12,7 +12,14 @@ endif::[]
 ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :repoURL: https://github.com/CS2103AUG2017-W13-B3/main/tree/master
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team W13-B3`      Since: `Jun 2016`      Licence: `MIT`
+
+Welcome to *DummyName*!
+
+*DummyName* is a desktop Address Book application, which is primarily a CLI (Command Line Interface)
```

Comment:
how about "contact management application" instead of "Address Book application"
Just a suggestion, not sure which is better.